.NET/Microsoft.Recognizers.Definitions/BasePhoneNumbers.cs

```diff
@@ -22,7 +22,7 @@ public static class BasePhoneNumbers
 		public const string BrazilPhoneNumberRegex = @"(((\B\(\s?))\d{2,3}(\s?\))|(\b\d{2,3}))\s?\d{4,5}-?\d{3,5}\b";
 		public const string UkPhoneNumberRegex = @"(((\b(00)|\B\+)\s?)?\b\d{2}\s?(\(0\))?\d{3,5}[/-]?\s?(\d{7,8}|\d{3}\s?\d{3})\b)";
 		public const string GermanyPhoneNumberRegex = @"(\b(\d{3,4}\s?[-/]?(\s?\d){7,8})\b)";
-		public const string USPhoneNumberRegex = @"((((\B\+)|\b)1(\s|-)?)|\b)?(\B\(|\b)\d{3}\)?\s?-?\s?\d{3}\s?-?\s?\d{4}\b";
+		public const string USPhoneNumberRegex = @"((((\B\+)|\b)\d{1,3}(\s|-)?)|\b)?(\(|\b)\d{2,3}\)?\s?-?\s?\d{3}\s?-?\s?\d{4,5}\b";
```

Comment:
The number in the referenced issue is not a US number, so I don't think a fix for it should change the US regex.
Perhaps add a new one? It's a common format in China and perhaps other places.